src/resolve.jl

```diff
@@ -197,8 +197,8 @@ function _run(io::IO, cmd::Cmd, args...; flags=String[])
 end
 
 function resolve(; force::Bool=false, io::IO=stderr, interactive::Bool=false, dry_run::Bool=false)
-    # if frozen, do nothing
-    if STATE.frozen
+    # if frozen or if backend is null, do nothing
+    if STATE.frozen || backend() == :Null
```

Comment:
When the backend is Null, we should also set `STATE.resolved = true`.

Reply:
I put this into the `backend()` function, which seems like the best place, no?